### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by gdh13**
minor updates; is AccountDock one of our companies?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,11 +56,14 @@ When making updates, run ```./utils/add_links``` to add links to GDPR cross-refe
...
+ * [Xplenty.com](https://xplenty.com)
+ * [DreamFactory.com] (https://www.dreamfactory.com/)
+ * [Iron.io] (https://www.iron.io/)
+ * [RingCaptcha.com] (https://ringcaptcha.com/)
  * [AccountDock.com](https://AccountDock.com)
 
 # License
 
-Copyright Jonathan Siegel, 2018-2019. Please contact cyber@xenon.io if you would like to re-use or contribute to this plan.
+Copyright Jonathan Siegel, 2018-2010. Please contact cyber@xenon.io if you would like to re-use or contribute to this plan.
```
Comment (usiegj00) [inline README.md]: Typo

**Review by usiegj00 — CHANGES_REQUESTED** [review]
Fix date and it's approved.

**Comment by gdh13** [comment]
Complete

**Comment by gdh13** [comment]
Corrections made; ready for approval

**Review by usiegj00 — APPROVED** [review]

**Review by usiegj00 — APPROVED** [review]
👍

**Pull request closed without merging**